service/service.go

```diff
@@ -115,13 +113,13 @@ func New() *Service {
 		listenPort = os.Getenv("LISTEN_PORT")
 	}
 
-	c := config.GetInstance()
+	Config().Name = name
```

Comment:
Can we check here if somebody tries to pass an empty string?
Just to be sure 🙂

Reply:
sure